tasks/merge/README.md

```diff
@@ -0,0 +1,7 @@
+# Merge task
+
+This task requires merging many small expression matrices into one large one.
+
+It addresses a particular problem for the HCA, which is merging many small SmartSeq2 "bundles" together, but more generally it involves performing relatively small reads from many different files.
```

Comment:
Better define the exact meaning of "bundles", from context it seems: one or more columns of expression profiles.

Reply:
I'll remove the reference to bundles. It probably only confuses things here.